tools/clang/unittests/HLSLExec/LongVectorTestData.h

```diff
@@ -289,7 +289,7 @@ INPUT_SET(InputSet::Bitwise, std::numeric_limits<int16_t>::min(), -1, 0, 1, 3,
           6, 9, 0x5555, static_cast<int16_t>(0xAAAA),
           std::numeric_limits<int16_t>::max());
 INPUT_SET(InputSet::SelectCond, 0, 1);
-INPUT_SET(InputSet::AllOnes, 1);
+INPUT_SET(InputSet::AllScalarOnes, 1);
```

Comment:
Isn't the "scalar" part implicit in all of these input sets?  What's special about this one that means we need to call them out?